.github/actions/install-aiida-core/action.yml

```diff
@@ -7,16 +7,18 @@ inputs:
     default: '3.9'              # Lowest supported version
     required: false
   extras:
-    description: aiida-core extras (including brackets)
-    default: ''
+    description: list of optional dependencies
+    # NOTE: The default 'pre-commit' extra recursively contains
+    # other extras needed to run the tests.
+    default: pre-commit
```

Comment:
I am a bit confuse here, how the extra like "atomic_tool" included in pre-commit extra?

Reply:
had the same issue last time, because pre-commit depends on them https://github.com/aiidateam/aiida-core/blob/02cbe0ceb7d9f9343911592e31abe6310d2bd38b/pyproject.toml#L214C13-L214C14